lambdas/enums/snomed_codes.py

```diff
@@ -0,0 +1,9 @@
+from enum import StrEnum
+
+
+class SnomedCodesType(StrEnum):
+    LLOYD_GEORGE = "16521000000101"
```

Comment:
Are we planning on adding more of these? Similar to the previous comment, if there aren't going to be any more codes we should look at making this a constant somewhere instead of a single-value Enum. If we're adding more in future, I'm happy for this to stay.